utilities/helpers/updateSpecificFieldsHelper.js

```diff
@@ -164,6 +165,23 @@ const processingParent = async (authorPermlink) => {
   await Wobj.update({ author_permlink: authorPermlink }, { parent: result.body });
 };
 
+const checkExistingTags = async (tagCategories = []) => {
+  for (const category of tagCategories) {
+    const existingTags = await redisGetter.getTagCategories(`tagCategory:${category.body}`);
+    const newTags = _
+      .filter(category.categoryItems, (o) => !_.includes(existingTags, o.name) && o.weight > 0);
+
+    if (!newTags.length) continue;
+    let counter = 0;
+    const tags = [];
+    for (let i = 0; i < newTags.length; i++) {
+      tags[counter++] = 0;
+      tags[counter++] = newTags[i].name;
+    }
+    await redisSetter.addTagCategory({ categoryName: category.body, tags });
```

Comment:
if we can add multiply data, you can call this method once in the end, with all tags